README.md

```diff
@@ -113,12 +115,34 @@ catkin do the following:
       ## TeXlipse
       .texlipse
       ```
+
+#### Multiple Packages
+The _eclipsify_ command can be used in conbitation with the catkin command to easily generate project files for all the ROS packages that  
+reside in a workspace. The options are as follows:
+
+  1. Create project files in the catkin worspace devel directory:
+    ```
+    catkin list -u | xargs -I pkg bash -c "eclipsify pkg"
+    ```
+    As in the single package example, the eclipse project files will be stored in a subdirectory in the **\<catkin workspace>/devel/share/..**  
+    directory.
+
+  2. Create project files in a separate __projects__ directory:
+    Run the following command from the root directory of your catkin workspace
+    ```
+    catkin list -u | xargs -I pkg bash -c "eclipsify pkg -O projects/pkg"
```

Comment:
Same here: `... | xargs eclipsify -O projects/pkg`.

Reply:
Not so sure on this one, The reason for the xargs **-I** option is to allow substituting the _pkg_ variable into the subsequent command.  Just running `xargs eclipsify -O projects/pkg` would not have the same effect.

Reply:
You are right it slightly changes how `eclipsify` gets called. My suggestion would move the packages after the `-O` option and its argument. But this is perfectly fine (thanks to the great python ArgumentParser) and even preferable IMO.